conditional/blueprints/slideshow.py

```diff
@@ -136,6 +136,7 @@ def slideshow_spring_review(user_dict=None):
     if status == "Passed":
         if ldap_is_intromember(account):
             ldap_set_not_intro_member(account)
+        ldap_set_housingpoints(account, ldap_get_housingpoints(account) + 2)
```

Comment:
future: do we have handling of success/failure in ldap operations to give evals/rtps a list at the end of "here's all the results that need to be fixed"?

Reply:
I don't think we have that anywhere. I think a new issue is in order for that